api/src/wfl/module/xx.clj

```diff
@@ -2,196 +2,177 @@
-            [clojure.set :as set]
             [clojure.string :as str]
-            [clojure.tools.logging :as log]
+            [clojure.tools.logging.readable :as logr]
+            [wfl.api.workloads :refer [defoverload]]
+            [wfl.api.workloads :as workloads]
+            [wfl.environments :as env]
+            [wfl.jdbc :as jdbc]
             [wfl.module.all :as all]
-            [wfl.references :as references]
-            [wfl.service.cromwell :as cromwell]
             [wfl.service.gcs :as gcs]
+            [wfl.service.postgres :as postgres]
+            [wfl.service.cromwell :as cromwell]
             [wfl.util :as util]
             [wfl.wdl :as wdl]
-            [wfl.wfl :as wfl]))
-
-(def description
-  "Describe the purpose of this command."
-  (let [in    "gs://daly_talkowski_asc_external/"
-        out   (str in "picard_out_hg38/")
-        title (str (str/capitalize wfl/the-name) ":")]
-    (-> [""
-         "%2$s eXternal eXomes"
-         "%2$s %1$s xx reprocesses BAMs or CRAMs to CRAMs."
-         ""
-         "Usage: %1$s xx <env> <max>"
-         "       %1$s xx <env> <in> <out>"
-         "       %1$s xx <env> <max> <in> <out>"
-         ""
-         "Where: <env> is an environment,"
-         "       <max> is a non-negative integer,"
-         "       and <in> and <out> are GCS urls."
-         "       <max> is the maximum number of inputs to process."
-         "       <in>  is a GCS url to files ending in '.bam' or '.cram'."
-         "       <out> is an output URL for the reprocessed CRAMs."
-         ""
-         "The 2-argument (<env> <max>) command releases up to <max>"
-         "workflows from 'On Hold' to 'Submitted' in the Cromwell"
-         "specified by the environment <env>."
-         ""
-         "The 3-argument command reports on the status of workflows"
-         "and the counts of BAMs and CRAMs in the <in> and <out> urls."
-         ""
-         "The 4-argument command queues up to <max> .bam or .cram files"
-         "from the <in> URL 'On Hold' to the Cromwell in environment <env>."
-         ""
-         (str/join \space ["Example: %1$s xx xx 9" in out])]
-        (->> (str/join \newline))
-        (format wfl/the-name title))))
+            [wfl.wfl :as wfl])
+  (:import [java.time OffsetDateTime]))
+
+(def pipeline "ExternalExomeReprocessing")
+
+(def cromwell-labels
+  "The WDL label applied to Cromwell metadata."
+  {(keyword wfl/the-name) pipeline})
 
 (def workflow-wdl
   "The top-level WDL file and its version."
-  {:release "ExternalExomeReprocessing_v1.1"
-   :top "pipelines/reprocessing/external/exome/ExternalExomeReprocessing.wdl"})
+  {:release "ExternalExomeReprocessing_v2.1.1"
+   :top     "pipelines/broad/reprocessing/external/exome/ExternalExomeReprocessing.wdl"})
 
-(def cromwell-label-map
-  "The WDL label applied to Cromwell metadata."
-  {(keyword (str wfl/the-name "-xx"))
-   (wdl/workflow-name (:top workflow-wdl))})
+(def reference-fasta-defaults
```

Comment:
Could you use this `reference_fasta` instead? https://github.com/broadinstitute/wfl/blob/master/api/src/wfl/references.clj#L4

Reply:
Sure.